alma/lib/Utils/Settings.php

```diff
@@ -135,6 +140,18 @@ public static function canLog()
         return (bool) (int) self::get('ALMA_ACTIVATE_LOGGING', false);
     }
 
+    public static function canShareOfCheckout()
+    {
+        return (bool) (int) self::get(ShareOfCheckoutAdminFormBuilder::ALMA_ACTIVATE_SHARE_OF_CHECKOUT, true);
+    }
+
+    public static function dateShareOfCheckout()
```

Comment:
```suggestion
    public static function getCurrentTimestamp()
```

In a generic Utils/Settings class, shareOfCheckout should not be known or used, only generic concepts here